install/coreos.md

```diff
@@ -13,8 +13,8 @@ These instructions are for CoreOS and VMWare Photon.
 
 To install and configure PX via the Docker CLI, use the command-line steps in this section.
 
->**Important:**<br/>PX stores configuration metadata in a KVDB (key/value store), such as Etcd or Consul. If you have an existing KVDB, you may use that.  If you want to set 
-one up, see the [etcd example](/run-etcd.html) for PX
+>**Important:**<br/>PX stores configuration metadata in a KVDB (key/value store), such as Etcd or Consul. If you have an existing KVDB, you may use that.  If you want to set
+one up, see the [etcd example](/maintain/etcd.html) for PX
```

Comment:
We must _always_ recommend customers to setup a dedicated etcd for PX and never recommend using an existing etcd cluster shared by other apps